es.hjson

```diff
@@ -432,56 +432,56 @@
-    "oneError.cantSendPost": "❌ <b>No podría enviar el mensaje, asegúrese de que %s sea un administrador con suficientes permisos en %s.</b>%s"
-    "oneSuccess": "✅ <b>¡Ok, publicado con éxito!</b>\n\nVer https://t.me/%s/%s para la publicación!\n\n<b>proteja a su grupo, debe revocar todos los enlaces de invitación anteriores. Puede hacerlo mediante:</b>\n1. Revocando su Invite Link personal\n2. Degradar y luego volver a promocionar a todos sus administradores (excepto CalsiBot). Esto revocará cualquier enlace de invitación que hayan compartido."
+    "oneError.targetIsNotChannel": "❌ <b>El enlace que ha proporcionado es para un Grupo público.\n\nPreVerify está definido para usarse solo con canales públicos.</b>"
+    "oneError.cantSendPost": "❌ <b>No he podido enviar el mensaje. Asegúrese de que %s sea un administrador con suficientes permisos en %s.</b>%s"
+    "oneSuccess": "✅ <b>¡Ok, publicado con éxito!</b>\n\nVer https://t.me/%s/%s para la publicación!\n\n<b>Para proteger completamente su grupo, debe revocar todos los enlaces de invitación anteriores. Puede hacerlo de la siguiente manera:</b>\n1. Revoca tu enlace de invitación personal\n2. Remueve y vuelve a promover a tus administradores nuevamente (excepto CalsiBot). Esto revocará cualquier enlace de invitación que hayan compartido."
   }
 
   "joinverify.preverify.imageText": {
     "title": "Para unirse al grupo:"
     "stepOne": "Toque <span>Verificar,</span> luego <span>Inicio</span>"
-    "stepTwo": "<span>Pase</span> la prueba"
+    "stepTwo": "<span>Pase</span> la verificación humana"
     "stepThree": "<span>¡Unirse</span> al grupo!"
   }
 
   "joinverify.preverify.main": "<b>%s%s está <a href=\"t.me/CalsiBotDev/111\">protegido por</a> PreVerify.</b>"
-  "joinverify.preverify.verificationMessage": "<b>Este mensaje es para verificar la propiedad de este canal para la función de PreVerify de CalsiBot.</b>\n\n%s"
+  "joinverify.preverify.verificationMessage": "<b>Este mensaje es para verificar la propiedad de este canal mediante la función de PreVerify por CalsiBot.</b>\n\n%s"
   "joinverify.preverify.setUpCancelled": "<b>Configuración de PreVerify cancelada.</b>"
 
 
   /* ---------------------------------------------------------------------
                         JoinLimit - Used for /JoinLimit
      --------------------------------------------------------------------- */
-  "joinlimit.joinevents.userLimited": "%s ha sido restringido %s porque <code>/JoinLimit</code><b> está habilitado.</b>"
-  "joinlimit.joinevents.userLimitedTimed": "%s ha sido restringido %s <b>hasta %s (UTC) porque </b><code>/JoinLimit</code><b> está habilitado</b>"
+  "joinlimit.joinevents.userLimited": "%s ha sido restringido %s porque <code>/JoinLimit</code><b> está activado.</b>"
+  "joinlimit.joinevents.userLimitedTimed": "%s ha sido restringido %s <b>hasta %s (UTC) porque</b><code>/JoinLimit</code><b>está activado</b>"

```

Comment:
These spaces are important, otherwise the text will look like:
> **porque**`/JoinLimit`**está activado**

```suggestion
  "joinlimit.joinevents.userLimitedTimed": "%s ha sido restringido %s <b>hasta %s (UTC) porque </b><code>/JoinLimit</code><b> está activado</b>"
```